packages/host/app/lib/externals.ts

```diff
@@ -92,4 +92,8 @@ export function shimExternals(virtualNetwork: VirtualNetwork) {
   });
   virtualNetwork.shimModule('super-fast-md5', superFastMD5);
   virtualNetwork.shimModule('matrix-js-sdk', matrixJsSDK);
+  virtualNetwork.shimAsyncModule({
+    id: 'uuid',
+    resolve: () => import('uuid'),
+  });
```

Comment:
we actually have a uuid module that is isomorphic if that is what this is all about. it's exported from runtime common here: https://github.com/cardstack/boxel/blob/main/packages/runtime-common/index.ts#L68. it works in both the browser and in node.